standards/model_description.ipynb

```diff
@@ -0,0 +1,614 @@
+      "text/plain": [
+       "<Figure size 1440x288 with 3 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "_, ax = plt.subplots(1, 3, figsize=(20, 4))\n",
+    "[ax[i].scatter(sample_data.time[i], sample_data.derivative[i]) for i in range(3)]\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As you can see algorithm removed some of the single distant outliers that are not informative."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Distribution of radial mode value for train and test data:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+   
```

Comment:
Используй, пожалуйста, grammarly. Или гугл-переводчик на нейросетях. Или пиши проще, прояви сочувствие к читателю :) Two models show approximately the same results but we recommend to use the second approach because it is more stable.

Reply:
Буду проявлять сочувствие:)
исправил